test/PowerPod.js

```diff
@@ -1,60 +1,154 @@
-const { expect, time, ether } = require('@1inch/solidity-utils');
+const { expect, time, ether, deployContract, constants } = require('@1inch/solidity-utils');
 const { ethers } = require('hardhat');
 const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
 const { expBase } = require('./helpers/utils');
 
 describe('PowerPod', function () {
-    let addr, addr1;
-    let accounts;
-    const commonLockDuration = time.duration.days('40');
-    const amountToStake = ether('1');
-
-    const stakeAndRegisterInDelegation = async (st1inch, delegation, user, amount, userIndex) => {
-        await st1inch.connect(user).deposit(0, commonLockDuration);
-        await st1inch.depositFor(user.address, amount);
-        await st1inch.connect(user).addPod(delegation.address);
-        await delegation
-            .connect(user)
-            .functions['register(string,string)'](
-                `${userIndex}DelegatingToken`,
-                `A${userIndex}DT`,
-            );
-        await delegation.connect(user).delegate(user.address);
-    };
-
     async function initContracts() {
-        const TokenPermitMock = await ethers.getContractFactory('ERC20PermitMock');
-        const oneInch = await TokenPermitMock.deploy('1inch', '1inch', addr.address, ether('200'));
-        await oneInch.deployed();
+        const commonLockDuration = time.duration.days('40');
+        const MAX_WHITELISTED = 3;
+        const BALANCE_THRESHOLD = 1000; // 10%
```

Comment:
Constants should be somewhere on top of the file.

Also `commonLockDuration => COMMON_LOCK_DURATION`.

Reply:
fixed